pkg/vpcmodel/abstractVPC.go

```diff
@@ -8,6 +8,8 @@ import (
 type VPCResourceIntf interface {
 	UID() string
 	Name() string
+	Names() (string, []string)
```

Comment:
having both Name() and Names() is strange. maybe change it to DisplayNames()

Reply:
np